### ISSUE TITLE
Advancement in image enhancement required

### ISSUE THREAD

**Issue by bvks2020sameer** [bvks2020sameer/Photolab#2]
**Describe alternatives you've considered**
Add histogram equalization and ai based photo solutions

**Comment by Manoj-2702** [comment]
@bvks2020sameer Shall I take up the issue by trying to add ai based photo solutions?

**Comment by bvks2020sameer** [comment]
Sure was looking for some ai advancements

**Comment by Manoj-2702** [comment]
Ok so what exactly do you want me to do ?

**Comment by bvks2020sameer** [comment]
just make a pull request . If possible please give a brief description of your code here . I'll review it and merge

**Comment by Manoj-2702** [comment]
Ok

**Comment by Manoj-2702** [comment]
We should enhance the image given rt?

**Comment by bvks2020sameer** [comment]
yes it would be a part of the Photolab operations. Photolab is a diverse framework starting from image enhancement to feature engineering.

**Comment by bvks2020sameer** [comment]
please do create a new issue before posting your code so that I can make you accountable for your contributions

**Comment by zacharias1219** [comment]
Hi I'm new to open source but well versed with ai and ml, what all do I need to do in order to contribute to this project.

**Comment by bvks2020sameer** [comment]
Hi , in project photolab, we are trying to build a cnn from scratch. So if there are any changes to be made or there are any new algorithms that can be added , you can share here and contribute to the repo.
Thankyou

**Comment by tarun2684** [comment]
Hi @bvks2020sameer ,

I’m new to open source and would love to contribute to this project! I saw that you're working on image enhancement and AI-based photo solutions as part of Photolab.

I’m comfortable with Python and have started learning about CNNs and image processing. 

I’d be happy to implement basic enhancements like histogram equalization, denoising, or even a simple CNN-based enhancement if that aligns with the goals.
